joint_trajectory_controller/src/joint_trajectory_controller.cpp

```diff
@@ -70,6 +71,9 @@ JointTrajectoryController::init(
   lifecycle_node_->declare_parameter<std::vector<std::string>>("joints", joint_names_);
   lifecycle_node_->declare_parameter<std::vector<std::string>>("write_op_modes", write_op_names_);
   lifecycle_node_->declare_parameter<double>("state_publish_rate", 50.0);
+  lifecycle_node_->declare_parameter<double>("action_monitor_rate", 20.0);
+  lifecycle_node_->declare_parameter<bool>("allow_partial_joints_goal", allow_partial_joints_goal_);
+  declareSegmentTolerances(lifecycle_node_);
```

Comment:
The two lines in that function could easily be placed here. I think they make more sense to be part of all declared parameters.

Reply:
fixed e12774f